spock/mcp/mcpacket_extensions.py

```diff
@@ -349,7 +348,7 @@ def decode_extra(packet, bbuff):
     def encode_extra(packet):
         bbuff = utils.BoundBuffer()
         if packet.data['nbt'] is None:
-            packet.data['nbt'] = nbt._TAG_End()
+            packet.data['nbt'] = nbt._TagEnd()
```

Comment:
Sure, just wanted to be compliant, supposing you really didn't wanted to make that class available (one underscore is meant for that). Checkout __all__ if that's the case!